src/PIL/MpoImagePlugin.py

```diff
@@ -95,6 +98,22 @@ def _close__fp(self):
         finally:
             self.__fp = None
 
+    @staticmethod
+    def adopt(jpeg_instance, mpheader=None):
+        '''
```

Comment:
Minor thing fort consistency:
```suggestion
        """
```